caleb-lab/test/note-routes-test.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

Off to a good start so far. Just get your test requirements in per the lab assignment and you should be good to go.